share/sysbuild/image_config.cmake

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# This sysbuild CMake file sets the sysbuild controlled settings as properties
+# on Zephyr images.
+
+get_target_property(image_board ${image} BOARD)
+if((NOT image_board) OR ("${image_BOARD}" STREQUAL "${BOARD}"))
+  get_target_property(${image}_APP_TYPE ${image} APP_TYPE)
+  if(NOT ${image}_APP_TYPE STREQUAL BOOTLOADER)
+    set_config_bool(${image} CONFIG_BOOTLOADER_MCUBOOT "${SB_CONFIG_BOOTLOADER_MCUBOOT}")
+    set_config_string(${image} CONFIG_MCUBOOT_SIGNATURE_KEY_FILE
+                               "${SB_CONFIG_BOOT_SIGNATURE_KEY_FILE}"
+    )
+  endif()
```

Comment:
Looking at this, `APP_TYPE` seems like something that should be extended to a more generic `GROUP` property. I think it would be useful to group images arbitrarily and set a common configuration for all images in (or not in) a given group.

It would also be nice to let one image belong to multiple groups and, eventually, to assign existing images to new groups defined out of tree.

Reply:
sounds like a good idea.

Will take a look.

Reply:
covered by this comment: https://github.com/zephyrproject-rtos/zephyr/pull/55871#discussion_r1202951853